### ISSUE TITLE
`1.0` release

### ISSUE THREAD

**Issue by martinszeltins** [victorgarciaesgi/regle#27]
Hey! I have been testing Regle and it looks really good. I would say more stable and with less bugs than Vuelidate. Thank you for fixing all the bugs.

What do you think about releasing version `1.0` stable? Currently it gives the idea that this is not to be used in production apps which is also what README says:

> ⚠️ Project is still in development, do not use it in production API or function names can still change

I would suggest releasing `1.0` and let people start using it for real, report any issues and help the project grow. I think most people see this warning and run away immediately which is a shame because this is shaping up to be a really good library. If we release `1.0` more people will join in testing and reporting bugs as well.

Perhaps the only real missing piece would be to improve the documentation which I want to send in a PR soon.  :)

But honestly, it feels pretty stable already! Awesome job! 🥳

**Comment by victorgarciaesgi** [comment]
Thanks! 😍 

Yep it's definitely planned, but I want to be the `1.0` to be robust. I'm still patching edge cases on my side.
Thanks for contributing to the docs! It's really helpful as it's not easy
I have a little roadmap before releasing the `1.0`.
### For v1.0

- [ ] Unit tests
- [ ] E2E tests
- [ ] Type tests
- [ ] Improve coverage
- [ ] Simplify/reduce core code
- [ ] Example repo, Reproduction repl

### Next iterations

- [ ] `Valibot` support
- [ ] `Yup` support
- [ ] TS docs
- [ ] `withErrorType`

**Comment by martinszeltins** [comment]
Awesome! Maybe a gift for Christmas 🎁

**Comment by victorgarciaesgi** [comment]
I'm just missing a advanced example repo and this would be good for release :D

**Comment by martinszeltins** [comment]
> I'm just missing a advanced example repo and this would be good for release :D

Awesome! 🥳  I am also currently working on a few more examples to add to the docs, something that I myself would find useful if I was a beginner. Will send in a pull request in the next few days.

**Comment by victorgarciaesgi** [comment]
So cool! Thank you 🫶🏻🫶🏻

**Comment by martinszeltins** [comment]
> I'm just missing a advanced example repo and this would be good for release :D

I'll tell all my friends about it when 1.0 is released 👼🏻

**Comment by victorgarciaesgi** [comment]
Yep I hope next week will do! I'm still notifying some edge cases while doing the advanced example, but i'll present it to my company monday morning too

**Comment by martinszeltins** [comment]
@victorgarciaesgi Hey, how close do you think we are to version 1.0? I see a lot of great work being done lately 🙏

**Comment by victorgarciaesgi** [comment]
Hey! Thanks 😍 

Testing it with success on production, it appear super stable now. I just have a feature still I need for the schemas here #55

**Comment by victorgarciaesgi** [comment]
Hey @martinszeltins Finally released the `1.0` 🎉  I will communicate over this tomorrow
